ts/Core/Renderer/SVG/SVGRenderer.ts

```diff
@@ -2474,6 +2471,20 @@ class SVGRenderer {
         );
 
     }
+
+    /**
+     * Re-align all aligned elements.
+     *
+     * @private
+     * @function Highcharts.SVGRenderer#align
+     * @return {void}
+     */
+    public align(): void {
```

Comment:
I think we should call this function `alignElements` to avoid (cognitive) confusion with `SVGElement.align` and the `align` property of SVGAttributes.